### DISCUSSION TITLE
大家好阿OvO

### DISCUSSION THREAD

**Discussion by Bochiii984** [share121/inter-knot#871] [其他]
大家好阿(o゜▽゜)o☆，这里是腾讯游戏Key社Summmerpockets的鸣濑白羽，一起期待25年SP的动画化吧╰(*°▽°*)╯
![06333e819fc5e8a29b102dbc777aa08482833c4b_raw](https://github.com/user-attachments/assets/f9c2aef4-c114-42bf-81f1-391c911d808d)
![0ab2a33a8eb51b31aab09b778bf63ead61b915b3](https://github.com/user-attachments/assets/4354625b-47fb-4e7f-8b6b-a6b6daf086ea)

**Comment by Lanxey** [comment]
https://www.bilibili.com/video/BV1Ws411J7Ah/

**Comment by Ikaros423** [comment]
白羽酱，你能表演一下那个吗，就是那个，（手比成枪的形状）“灵弹！”

**Reply by Bochiii984** [reply]
![202305220203413415db8c692-15ba-41d5-85ad-eaa5b8320da7](https://github.com/user-attachments/assets/a18680f3-6620-461d-b86b-213657f2b9d9)

**Reply by Ikaros423** [reply]
呜啊，我倒_(:з」∠)_

**Comment by warning-Kirov-reporting** [comment]
腾讯游戏？这不是绳网上最大的二次元游戏公司吗
